packages/orchestrator-ui-components/src/components/WfoTable/WfoTableWithFilter/updateQueryString.ts

```diff
@@ -0,0 +1,52 @@
+const getValueWithoutBrackets = (value: string) => {
+    if (value.startsWith('(') && value.endsWith(')')) {
+        return value.slice(1, -1);
+    }
+    return value;
+};
+
+const isContainingSpaces = (value: string) => {
```

Comment:
Nitpicking but hasSpaces also confirms to the verbNoun pattern of naming

Reply:
Done